ltac/g_class.ml4

```diff
@@ -68,9 +68,9 @@ END
 (** Compatibility: typeclasses eauto has 8.5 and 8.6 modes *)
 TACTIC EXTEND typeclasses_eauto
  | [ "typeclasses" "eauto" "bfs" int_or_var_opt(d) "with" ne_preident_list(l) ] ->
-    [ typeclasses_eauto ~strategy:Bfs ~depth:d l ]
+    [ typeclasses_eauto ~allow_shelved:true ~strategy:Bfs ~depth:d l ]
  | [ "typeclasses" "eauto" int_or_var_opt(d) "with" ne_preident_list(l) ] ->
-    [ typeclasses_eauto ~depth:d l ]
+    [ typeclasses_eauto ~allow_shelved:true ~depth:d l ]
```

Comment:
Fixed now, they don't specify the allow_shelved flag anymore, taking their value from the global setting.